check-code/Makefile

```diff
@@ -0,0 +1,5 @@
+test: .cabal-sandbox
```

Comment:
Is `.cabal-sandbox` really the right way of checking whether anything needs to be done?